classes/item.rb

```diff
@@ -12,6 +12,11 @@ def initialize(publish_date, archived)
     @genre = nil
   end
 
+  def add_label(label)
```

Comment:
I know this is probably an oversight but it's worth mentioning, the add_label method is a setter for the label instance variable.

You can rewrite it thus

`def label=(label) ........  end`

Good work on your implementation details

Reply:
👍